documentation/docs/how_tos/outputs_and_transactions/01_send_transaction.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Send a Transaction
+description: 'How to send a value transaction using wallet.rs'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- java
+- nodejs
+- python
+- rust
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from "../../libraries/nodejs/how_to/outputs_and_transactions/_05_send_transaction/_code_sample.mdx";
+import NodejsOutput from "../../libraries/nodejs/how_to/outputs_and_transactions/_05_send_transaction/_expected_output.md";
+import PythonCode from "../../_admonitions/_guide_coming_soon.md";
+import PythonOutput from "../../_admonitions/_guide_coming_soon.md";
+import RustCode from "../../libraries/rust/how_to/outputs_and_transactions/_05_send_transaction/_code_sample.mdx";
+import RustOutput from "../../libraries/rust/how_to/outputs_and_transactions/_05_send_transaction/_expected_output.md";
+import TipFaucet from "../../_admonitions/_tip_faucet.md";
+import IOTAJsExample from "../../_admonitions/_info_iotajs.mdx";
+
+Once you have [created your account](../accounts_and_addresses/01_create_a_wallet_account.mdx) and
+[requested funds from the faucet](../accounts_and_addresses/03_request_funds.mdx) you are ready to send your first
+transaction.
+
+<TipFaucet/>
+
+<IOTAJsExample exampleURL={"https://wiki.iota.org/iotajs/how_tos/transaction"}/>
```

Comment:
Why do we link to the iota.js client library here?

Reply:
These links were added so that we can give [comms a unified link for certain CTAs they want to promote ](https://docs.google.com/spreadsheets/d/1AJ_XenrOwpoXfmmrt5UbSWYyQKqMrCF_spMK6hPzSFM/edit#gid=928771981).  Since including the actual iota.js example here would either tarnish the wallet.rs repo, or be really complicated docusaurus-wise, this seemed like a nice workaround.

Reply:
Hm okay, just not sure why iota.js should be there, since that is a client library and not a wallet library
One could also link to iota.rs then (please don't also add that :trollface:)
But feel free to keep it as it is